index.html

```diff
@@ -30,6 +30,23 @@
         </ul>
       </ol>
     </div>
+    <div id="ten_list">
+      <ul>
+        <li>Top Ten Abilities</li>
```

Comment:
Typically wouldn't want to use an li for the heading here, as it's not explicitly readable. Could get rid of the ul wrapping this altogether, and just make 'Top Ten Abilities' an <h2> or <h3>.

Might also be a good idea to attach a class/id to the heading here to manipulate the <li>s inside more easily.